### PR TITLE
Rename review-period-end to last-call-deadline

### LINKED ISSUE #4413: Rename `review-period-end` to `last-call-end` or `last-call-deadline`
It is confusing when the `Review` state doesn't require it, but the `Last Call` state does.  It should have been changed when the `Review` state (at last) was introduced.

### PR DESCRIPTION
**Pull request by axic**
Closes #4413

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ matrix:
     - language: rust
       cache: cargo
       before_script:
-        - cargo install eipv --version=0.3.0
+        - cargo install eipv --version=0.4.0
```
Comment (axic) [inline .travis.yml:33]: @lightclient can you make this release please? :)

**Review by MicahZoltu — APPROVED** [review]
Depending on how long it takes to cut a release of the EIP validator, we probably will need to do one more pass of changes of `review-period-end` before merging.

**Comment by lightclient** [comment]
@MicahZoltu it's been cut! https://crates.io/crates/eipv/0.4.0

**Comment by axic** [comment]
@MicahZoltu needs a manual merge now

**Pull request merged**